item-seller/readme.md

```diff
@@ -1,4 +1,7 @@
-This guide will walk you through the process of building contracts, deploying them into an existing world running in Docker, and testing their functionality by executing scripts.
+## Introduction
+This guide will walk you through the process of building contracts for the item seller, deploying them into an existing world running in Docker, and testing their functionality by executing scripts.
+
+The Item Seller allows players to purchase items with ERC20 Tokens.
```

Comment:
Can we add more details here please ? 
eg: A SSU in game can act like a itemSeller with ...
What happens inside a item seller. We should explain the concept of inventory , ephemeral inventory and details about it

Reply:
Added more explanation, and a step by step on how it works.